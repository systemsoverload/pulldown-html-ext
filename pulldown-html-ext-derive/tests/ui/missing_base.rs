use pulldown_html_ext_derive::html_writer;

struct HtmlWriterBase<W>(W);

#[html_writer]
struct MissingBase<W> {  // This should fail
    writer: W,  // Wrong field name
}

fn main() {}
