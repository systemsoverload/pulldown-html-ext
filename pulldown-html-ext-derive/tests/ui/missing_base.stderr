error: struct must have a field named 'base'
 --> tests/ui/missing_base.rs:6:23
  |
6 |   struct MissingBase<W> {  // This should fail
  |  _______________________^
7 | |     writer: W,  // Wrong field name
8 | | }
  | |_^
